eslint.config.mjs

```diff
@@ -0,0 +1,80 @@
+import path from "node:path"
+import tseslint from "typescript-eslint"
+import url from "node:url"
+import unusedImports from "eslint-plugin-unused-imports"
+
+const __dirname = path.dirname(url.fileURLToPath(import.meta.url))
+
+export default tseslint.config(
+    // register plugins
+    {
+        plugins: {
+            ["@typescript-eslint"]: tseslint.plugin,
+            ["@unused-imports"]: unusedImports,
+        },
+    },
+
+    // add files and folders to be ignored
+    {
+        ignores: [
+            "**/*.js",
+            "**/*.spec.ts",
+            "eslint.config.mjs",
+            "vitest.config.ts",
+            ".github/*",
+            ".yarn/*",
+            ".testing/*",
+            "dist/*",
+            "docs/*",
+            "scripts/*",
```

Comment:
why don't you want to check the scripts?

Reply:
```
/Users/petrmakhnev/tvm-dec2/scripts/gen-codepage.ts
  0:0  error  Parsing error: /Users/petrmakhnev/tvm-dec2/scripts/gen-codepage.ts was not found by the project service. Consider either including it in the tsconfig.json or including it in allowDefaultProject

✖ 1 problem (1 error, 0 warnings)

```